types/database/models/servers/playbook_run_model.ts

```diff
@@ -61,6 +61,9 @@ declare class PlaybookRunModel extends Model {
     // Timestamp of the last status update
     last_status_update_at: number;
 
+    // Timestamp of the last run update
+    last_update_at: number;
```

Comment:
isn't this the same as last_status_update_at?

Reply:
seems to be the same and it also matches some array in the response about timeline events, so I guess this one is not needed.